docs/python/1.6/python-appapi-devguide-2.md

```diff
@@ -9,43 +9,21 @@ prev:
   file: python-appapi-devguide
   title: Developing IBM Streams Applications with Python
 next:
-  file: python-appapi-devguide-3
-  title: 2.0 Developing your first application
+  file: python-appapi-devguide-2a
+  title: 2.0 Developing for the Streaming Analytics service
```

Comment:
The titles for the two sections are a bit incorrect.
"Developing for the Streaming Analytics service" and "for Streams on-premises" make it sound mutually exclusive.  Really, though, the core applications themselves developed in the same way.  The difference is how you submit them and how you configure your environment.  So perhaps "developing without installing streams"  and "developing with a streams install" work better.